packages/virgo/src/utils/transform-input.ts

```diff
@@ -10,35 +10,26 @@ function handleInsertText<TextAttributes extends BaseTextAttributes>(
 ) {
   if (!data) return;
   editor.insertText(vRange, data, attributes);
-  editor.setVRange(
-    {
-      index: vRange.index + data.length,
-      length: 0,
-    },
-    false
```

Comment:
Why we use true here?

Reply:
To trigger vertical / horizontal scroll on text input.
e.g. 
- on adding text / paragraphs,  page should scroll vertically.
- inside embedded code block to have horizontal scroll

earlier these scrolls was triggered indirectly by selection change event which is async.
text will be added to browser -> selection change event -> scroll

now scroll is being triggered on only on sync vrange changes.